pyproject.toml

```diff
@@ -1,3 +1,13 @@
+[build-system]
+requires = [
+  "pip >= 19.3.1",
+  "setuptools >= 41.4.0",
+  "setuptools_scm >= 3.3.3",
+  "setuptools_scm_git_archive >= 1.1",
```

Comment:
why do we need `pip`, `setuptools_scm `  and `setuptools_scm_git_archive ` here?

Reply:
I am sure pip is needed because otherwise you expose it to versions of pip that are not able to build it. Keep in mind that this is the pip used for package building, not the one used to install it (an already builded wheel does not need any of these).

I will remove the other two because they are needed only by projects that use scm based versioning, which dogpile.cache does not.

Reply:
@CaselIT Yep, these are not really install requirements like we used to have on setup.cfg, they are used in isolated environment and only when a wheel is not already build. This gives us more flexibility in controlling them.

Apparently installing from source using these passed at https://zuul.opendev.org/t/openstack/build/8393962aacd24ff286496dced54d5e44/console